csharp/src/Google.Protobuf/Google.Protobuf.nuspec

```diff
@@ -5,20 +5,34 @@
     <title>Google Protocol Buffers C#</title>
     <summary>C# runtime library for Protocol Buffers - Google's data interchange format.</summary>
     <description>See project site for more info.</description>
-    <version>3.0.0-beta*</version>
+    <version>3.0.0-alpha4</version>
```

Comment:
Also, from gRPC C# perspective, it would be much easier if we didn't have to append the prerelease suffix. To give you some background:
Grpc.Core package has been released as 0.5.0 and 0.6.0 so it's not treated as a "prerelease" by nuget. Other Grpc packages use the same convention and some of them need to depend on protobuf runtime. If we append -betaX or -alphaX, I would need to change Grpc package version to accomodate that somehow. And I can't really come with solution that I'd like as a whole.
Thoughts?